lib/phlex/translation.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Phlex
+	module Translation
+		def self.included(view)
+			view.extend(ClassMethods)
+		end
+
+		module ClassMethods
+			attr_writer :translation_path
+
+			def translation_path
+				@translation_path ||= name&.split("::")&.join(".")&.downcase.to_s
```

Comment:
This will use full class name, for example:
`Views::Layouts::Navbar` => `views.layouts.navbar`. I think it's fine to use `views.` prefix (later we can add a setting to ignore it), but the user has the option to do this at application.rb `config.autoload_paths << "#{root}/app/views"` instead `config.autoload_paths << "#{root}/app"`, and define Phlex views without `Views` module like it did in my project: 
- https://github.com/stephannv/nindika/blob/main/config/application.rb#L25
- https://github.com/stephannv/nindika/blob/main/app/views/layouts/navbar_component.rb

Ps.: 
- `sub` and `str["::"] = "."` didn't work here because it replaces just the first occurrence.
- `split("::").join(".")` is faster than `gsub("::", ".")` according to benchmark I did in my machine.
<img width="713" alt="Screen Shot 2022-10-22 at 12 21 50" src="https://user-images.githubusercontent.com/3025661/197347288-5b317489-400f-41f4-b6b8-12f415a8bbea.png">